README.rst

```diff
@@ -157,6 +157,41 @@ Lost mode is slightly different to the "Play Sound" functionality in that it all
 >>> message = 'Thief! Return my phone immediately.'
 >>> api.iphone.lost_device(phone_number, message)
 
+Find My Friends
+==============
+
+Once you have successfully authenticated, you can find your friends' locations!
+
+Locations
+********
+
+Returns the your friends' last known locations. You must install and initialize the Find My Friends app on an iOS device, signing in with your credentials. In addition, your friends must be sharing their location with your account using the Find My Friends app.
+
+>>> api.friends.locations
+
+Followers
+********
+
+Returns a list of your friends who you share your location with.
+
+>>> api.friends.followers
+
```

Comment:
From #265 PR : >>> contact_id = following['id']

Reply:
Yes. This is true -- and I thought obvious -- but does it require explicitly noting?

Reply:
When I am seeing "issues" that are support questions, sometimes stuff like that, yes it's needed.

That's why I also want to add the complete data like explained here https://github.com/picklepete/pyicloud/pull/160#discussion_r404048763 because people will see the result of their query.

Or you can add a Friend class with properties.

Reply:
Unfortunately, the data returned in `following` is different from `contactDetails` and slightly different from `followers`. It will take time to remove personal info from the full response. I can add a redacted response later given the ask here.

Not a huge fan on documenting Apple's private API given that it is likely to change in the future. 

related xckd: https://xkcd.com/1172/